### ISSUE TITLE
Incorrect docstrings in test_usage.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6444]
|<img alt="stephsolis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| stephsolis reported|
|-|-|
|Trac ID|trac#6444|
|Type|defect|
|Created|2013-04-18 01:38:26Z|

Docstrings in ```twisted/test/test_usage.py``` for ```ParseCorrectnessTest```, ```TypedTestCase```, and ```WrongTypedTestCase``` indicate that ```Options.parseArgs``` is being tested, but really ```Options.parseOptions``` is being tested.

Attachments:

* [ticket6444_v1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5d7/5d79cd9c687577795a2349d2470ab268da2af699/a9fb242f8181fb7fbd552192a1878ccb0073786a.diff) (1064 bytes) - added by stephsolis on 2013-04-18 01:42:22Z - 
* [ticket6444_v1.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5d7/5d79cd9c687577795a2349d2470ab268da2af699/fd3b084e0fdd722544eb4639fc4a3a6ff5c11258.patch) (1064 bytes) - added by stephsolis on 2013-04-18 01:47:49Z - change extension to .patch (d'oh!)
* [ticket6444_v2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5d7/5d79cd9c687577795a2349d2470ab268da2af699/5252f28d922d629bc677cb699bf40181c885b0e6.patch) (995 bytes) - added by stephsolis on 2013-04-19 03:25:04Z - Added epytext link markup
* [6444v3.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/5d7/5d79cd9c687577795a2349d2470ab268da2af699/65948e8e478c2afb96c949cb03702f2c41da3d7b.patch) (6670 bytes) - added by stephsolis on 2013-05-05 02:44:42Z - Addressed review suggestions

<details><summary>Searchable metadata</summary>

```
trac-id__6444 6444
type__defect defect
reporter__stephsolis stephsolis
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1366249106000000 1366249106000000
changetime__1367741458000000 1367741458000000
version__None None
owner__stephsolis stephsolis

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="stephsolis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|stephsolis removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_3"></a>|@wallrj set owner to stephsolis|
|-|-|

Thanks stephsolis. Well spotted.

 1. Your patch has trailing whitespace
 2. Missing news file
 3. Why don't you also add some of the missing docstrings eg to SubcommandTestCase and HelpStringTest etc

Please answer or address the points above and submit another patch for review.

**Comment by twisted-trac** [comment]
|<img alt="stephsolis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|stephsolis removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_5"></a>|@wallrj set owner to stephsolis|
|-|-|

Thanks stephsolis. 

The latest patch looks fine.

But I think on second thoughts I'll change the news file to a .misc - changes to test docstrings don't really need to show up in the next release notes.

Then I'll apply it to trunk.

-RichardW.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_6"></a>|@wallrj set status to `closed`|
|-|-|

(In [38347]) Apply 6444v3.patch: Improvments to the test_usage docstrings

Author: stephsolis
Reviewer: rwall
Fixes: [#6444](https://github.com/twisted/twisted/issues/6444)

Improvments to the test_usage docstrings.
